labs/3L.md

```diff
@@ -0,0 +1,108 @@
+
+# Auto-grader Instructions
+
+## Update Auto-grader
```

Comment:
Don't need to update.

But we can include this anyway, use `umm update` instead for cargo install